genai-perf/genai_perf/record/record.py

```diff
@@ -90,16 +91,7 @@ class Record(metaclass=RecordType):
     records
     """
 
-    def __init__(self, value: Union[float, int], timestamp: int):
-        """
```

Comment:
Why are we getting rid of the entire docstring?

Reply:
Because they're redundant when variables are typed and well named